Client/metrics.yaml

```diff
@@ -525,6 +525,47 @@ qr_code:
       - firefox-ios@mozilla.com
     expires: "2021-07-01"
 
+# Default browser card metrics
+default_browser_card:
+  dismiss_pressed:
+    type: counter
+    description: |
+      Counts the number of times default browser card is dismissed.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1644846
```

Comment:
This should be your github issue

Reply:
So, in this case we could use the following. 
https://github.com/mozilla-mobile/firefox-ios/issues/7151